pkg/handler/v1/project/project.go

```diff
@@ -85,6 +88,58 @@ func CreateEndpoint(projectProvider provider.ProjectProvider, privilegedProjectP
 	}
 }
 
+func createProjectByServiceAccount(saEmail string, projectReq projectReq, memberMapper provider.ProjectMemberMapper, userProvider provider.UserProvider, memberProvider provider.PrivilegedProjectMemberProvider, projectProvider provider.ProjectProvider) (*apiv1.Project, error) {
+	var humanUserOwnerList []*kubermaticapiv1.User
+	bindings, err := memberMapper.MappingsFor(saEmail)
+	if err != nil {
+		return nil, common.KubernetesErrorToHTTPError(err)
+	}
+
+	if len(bindings) == 0 {
+		return nil, errors.New(http.StatusInternalServerError, fmt.Sprintf("no bindings for service account user %s", saEmail))
+	}
+	saBinding := bindings[0]
+	if !strings.HasPrefix(saBinding.Spec.Group, rbac.ProjectManagerGroupNamePrefix) {
+		return nil, errors.New(http.StatusForbidden, "the Service Account is not allowed to create a project")
+	}
+	if len(projectReq.Body.Users) == 0 {
+		return nil, errors.New(http.StatusBadRequest, "expected user emails")
```

Comment:
you could change the message here to something like:

"owner user email list required for project creation through Service Account"

Reply:
done